idaes/core/base/reaction_base.py

```diff
@@ -214,7 +214,15 @@ def _validate_property_parameter_units(self):
         """
         r_units = self.get_metadata().default_units
         prop_units = self.config.property_package.get_metadata().default_units
-        for u in r_units:
+        for u in [
+            "time",
+            "length",
+            "mass",
+            "amount",
+            "temperature",
+            "current",
+            "luminous_intensity",
+        ]:
```

Comment:
Can we add a `fundamental_units` list to the `UnitSet` so we don't need this explicit embedded constant here?